src/bindings/python/src/openvino/runtime/opset6/ops.py

```diff
@@ -124,17 +124,66 @@ def mvn(
     return _get_node_factory_opset6().create("MVN", inputs, attributes)
 
 
+@singledispatch
 @nameable_op
-def read_value(init_value: NodeInput, variable_id: str, name: Optional[str] = None) -> Node:
+def read_value(init_value: NodeInput,
+               variable_id: str,
+               variable_type: Optional[Union[NumericType, Type, str]] = None,
+               variable_shape: Optional[TensorShape] = None,
+               name: Optional[str] = None) -> Node:
```

Comment:
@itikhono from the point of API compatibility it would be better to have `name` before added arguments.
From the other hand it looks more logical how it is done right now.
If it isn't critical, I vote for keeping it as is.